components/d2l-quick-eval/behaviors/d2l-quick-eval-siren-helper-behavior.js

```
@@ -86,17 +86,35 @@ D2L.PolymerBehaviors.QuickEval.D2LQuickEvalSirenHelperBehaviorImpl = {
 			.then(function(e) {
 				if (e && e.entity && e.entity.properties) {
 					const p = e.entity.properties;
-					const empty = [];
 					const publishAll = this._getAction(e.entity, 'publish-all-feedback');
-					const submissionListHref = this._getRelativeUriProperty(e.entity, empty);
 
+					const submissionListSubEntity = e.entity.getSubEntityByRel(Rels.Assessments.submissionApplication);
+					let submissionListHref = '';
+					if (submissionListSubEntity.properties && submissionListSubEntity.properties.path) {
+						submissionListHref = submissionListSubEntity.properties.path;
+					}
+
+					const evaluateAllSubEntity = e.entity.getSubEntityByRel(Rels.Assessments.assessAllApplcation);
```

`assessAllApplcation` should be `assessAllApplication`

---

also me and @mtjandra notices that `e.entity.getSubEntityByRel` fails silently (in other words when `e.entity.getSubEntityByRel` is given an undefined value, it still returns a sub-entity). is there a way we can change this?

---

Yeah I noticed that before as well. Probably something worth changing, but not in scope here.

---

Yes I would say this change is out of scope, there may be other component who maybe relies on this behavior? Who knows.